mlflow/pyfunc/model.py

```diff
@@ -281,6 +283,8 @@ def _save_model_with_class_artifacts_params(
         mlflow_model: The model to which to add the ``mlflow.pyfunc`` flavor.
         model_config: The model configuration for the flavor. Model configuration is available
             during model loading time.
+        model_code_path: The path to the code that is being logged as a PyFunc model. Can be used
+            to load python_model when python_model is None.
```

Comment:
Nit - This will improve the rendering on mlflow.org:

```suggestion
        model_code_path: The path to the code that is being logged as a PyFunc model. Can be used
            to load ``python_model`` when ``python_model`` is ``None``.
```

Reply:
@annzhang-db Can we provide more information about what the Python script is supposed to contain in the docstring? 

Can we also link to an example / link to the mlflow.org documentation? For the linking part, feel free to file a follow-up ticket for internal tracking and let me know once it's been filed